src/helics/helics_enums.h

```diff
@@ -238,7 +236,9 @@ typedef enum {
     /** registration has failed */
     HELICS_ERROR_REGISTRATION_FAILURE = -1,
     /** the function executed successfully */
-    HELICS_OK = 0
+    HELICS_OK = 0,
+    /** user system abort to match typical SIGINT value*/
+    HELICS_ERROR_USER_ABORT = 2
```

Comment:
The signal value for SIGINT is 2, but the exit code for signal values is calculated as 128+n for POSIX. Typical meaning for exit code 2 in POSIX systems is misuse of a shell built-in.

```suggestion
    HELICS_ERROR_USER_ABORT = 130
```